Readme.md

```diff
@@ -38,3 +38,26 @@ curl --include \
    --header "Sec-WebSocket-Version: 13" \
    http://localhost:8888/ws
 ```
+
+## Roadmap
+
+- [ ] Framing
```

Comment:
As i understand it, i think framing is supported to the extent that the RFC specifies it, however any extension can have it's own framing.

Reply:
ok. thx